Wikipedia/Code/MWKSiteInfo.h

```diff
@@ -12,8 +12,13 @@ NS_ASSUME_NONNULL_BEGIN
 /// Raw title for the receiver's main page.
 @property (readonly, copy, nonatomic) NSString *mainPageTitleText;
 
+@property (readwrite, copy, nonatomic) NSNumber *readingListsConfigMaxEntriesPerList;
```

Comment:
Looks like `readingListsConfigMaxEntriesPerList` and `readingListsConfigMaxListsPerUser` should be `readonly` here.